crates/mdk-uniffi/src/python/__init__.py

```
@@ -0,0 +1,13 @@
+"""MDK Python bindings - Marmot Development Kit"""
+try:
+    from .mdk_uniffi import * # noqa: F401
```

Ruff (0.14.6)

`from .mdk_uniffi import *` used; unable to detect undefined names

(F403)